init_autoloader.php

```diff
@@ -4,20 +4,6 @@
     $loader = include 'vendor/autoload.php';
```

Comment:
@gianarb 

I would even go as far as to just include `vendor/autoload.php` in `public/index.php` instead of including this file.

What do you think?